Test/PublishProGetUniversalPackage.Tests.ps1

```diff
@@ -355,6 +373,19 @@ Describe 'Publish-WhiskeyProGetUniversalPackage.when user specifies files to pub
     ThenPackageNotPublished 'myfile2.upack'
 }
 
+Describe 'Publish-WhiskeyProGetUniversalPackage.when user specifies files to publish and excluding some' {

```

Comment:
Should we add a warning to the task if the exclude wildcard pattern ends up matching all files and nothing gets published?